spec/system/page_objects/components/composer.rb

```diff
@@ -73,6 +73,10 @@ def has_content?(content)
         composer_input.value == content
       end
 
+      def has_value?(value)
```

Comment:
With this `has_value?` having an `expect`, we get a much nicer output.

I tried replacing `has_content?` with it, but it didn't seem to be respected when used in an `expect(composer).to have_content(...)` expression.